### PR TITLE
extend pip install info

### PR DESCRIPTION
**Pull request by Borda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,84 +58,84 @@ Lightning sets up all the boilerplate state-of-the-art training for you so you c
...
-        # OPTIONAL
-        return DataLoader(MNIST(os.getcwd(), train=True, download=True, transform=transforms.ToTensor()), batch_size=32)
-
-    @pl.data_loader
-    def test_dataloader(self):
-        # OPTIONAL
-        return DataLoader(MNIST(os.getcwd(), train=True, download=True, transform=transforms.ToTensor()), batch_size=32)
-```
-
+    ```python
```
Comment (williamFalcon) [inline README.md]: i don't understand why this was deleted and re-added? doesn't seem to add anything to the readme...

Comment (Borda) [reply README.md]: this is just a bug/issue of visualisation in github..

**Comment by Borda** [comment]
@williamFalcon ^^

File: README.md [hunk 2]
```diff
@@ -57,84 +58,84 @@ Lightning sets up all the boilerplate state-of-the-art training for you so you c
...
+    import os
```
Comment (williamFalcon) [inline README.md]: i don't have too much time to check this example. can we make this PR not change this whole block of code?

Comment (Borda) [reply README.md]: if you look, you can see that the example is equal as before, just shifted by one tab right to belong to the enumerated item

**Pull request merged**